tema-2-html.html/pizza.html

```diff
@@ -0,0 +1,60 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width">
+        <title>Pizza</title>
+    </head>
+    <body>
+        <form>
+         <fieldset>
+           <legend>Pizza Order</legend>
+                <label for="Your Name">Your Name</label>
+                   <input request type="text" name="Your Name" id="Your-Name" placeholder="Your Name">
```

Comment:
- pentru a face field-ul obligatoriu de completat, atributul se numeste `required`
- ar fi trebuit sa mai adaugi si `minlength` si `maxlength` pentru regulile de lungime a textului

```suggestion
                   <input required type="text" name="Your Name" id="Your-Name" placeholder="Your Name" minlength="3" maxlength="10">
```